lms/services/jstor.py

```diff
@@ -0,0 +1,28 @@
+from lms.views.helpers import via_url
+
+
+class JSTORService:
+    def __init__(self, settings):
+        self._settings = settings
+
+    @property
+    def enabled(self):
+        return self._settings.get("enabled", False)
+
+    @staticmethod
+    def via_url(request, document_url):
+        return via_url(
+            request,
+            document_url,
+            content_type="pdf",
+            options={"jstor.ip": request.registry.settings["jstor_ip"]},
```

Comment:
It's now a trivial change to do `{"jstor.ip": settings["jstor_ip"]}` to use a per application instance value.

Left the value from the env variable now as is probably easier to handle while we only have one.